plugins/org.eclipse.fordiac.ide.application/src/org/eclipse/fordiac/ide/application/figures/FBNetworkConnection.java

```diff
@@ -129,20 +132,45 @@ public void updateConLabels() {
 		}
 	}
 
+	public boolean hasTypeMismatch() {
```

Comment:
please don't use own error checks. This is very limiting. use the GraphicalAnnotationStyles.updateAnnotationFeedback()